executor/engine_env_getter.yml

```diff
@@ -17,7 +17,8 @@
 - name: append envNameList from ChaosEngine
   set_fact:
     envNameList: "{{ envNameList + [c_engine_name.stdout] }}"
+    envdict: "{{envdict | combine({c_engine_name.stdout : c_engine_value.stdout})}}"
 
 - name: append envValueList from ChaosEngine
   set_fact:
-    envValueList: "{{ envValueList + [c_engine_value.stdout] }}"
\ No newline at end of file
+    envValueList: "{{ envValueList + [c_engine_value.stdout] }}"
```

Comment:
We can remove it later. As of know i keep it because we can compare the envs of engine and experiments in logs to check whether override is working or not.